src/plugins/intel_gpu/src/graph/graph_optimizer/basic_memory_dependencies.cpp

```diff
@@ -36,7 +36,7 @@ void basic_memory_dependencies::run(program& p) {
         }
 
         if (node->get_preferred_impl_type() == impl_types::onednn
-            && (node->is_type<convolution>() || node->is_type<deconvolution>())) {
+            && (node->is_type<convolution>() || node->is_type<deconvolution>() || node->is_type<fully_connected>())) {
```

Comment:
what about gemm or reduce? isn't it necessary to check gemm or reduce for such case?
In addition, isn't it necessary to confirm that other node does not have sum post operation?

Reply:
I removed the whitelist for sum post op fusion. Thank you.